### PR TITLE
My new branch

### PR DESCRIPTION
**Pull request by mwmentor**
Scheduling BC for review.

### CODE REVIEW

File: docs/refarch/boundedContexts/accountLookupAndDiscovery/index.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Account Lookup And Discovery BC
```
Comment (mdebarros) [inline docs/refarch/boundedContexts/accountLookupAndDiscovery/index.md:1]: @koekiebox please review these changes to the Account Lookup & Discovery BC.

File: docs/refarch/boundedContexts/accountLookupAndDiscovery/index.md [hunk 2]
```diff
@@ -55,49 +57,12 @@ Get transfer/s Participant/s details.
...
 #### Description
 
-Get transfer/s Party/s details.
+Where a participant DFSP queries another participant DFSP for the details of a Party which the second DFSP holds, this UC is used to validate the request and provide the requested Party data to the requesting DFSP.
 
 #### Flow Diagram
 
 ![Use Case - Get Party](./assets/aldGetParty_20210825.png)
 >
-## Canonical Quote Model
```
Comment (mdebarros) [inline docs/refarch/boundedContexts/accountLookupAndDiscovery/index.md:64]: Was this moved somewhere else?

File: docs/refarch/boundedContexts/auditing/index.md [hunk 3]
```diff
@@ -8,11 +8,11 @@ The Auditing BC is responsible for maintaining an immutable record of all of the
 * Key Management System
 * Cryptographic[^1] Provider Modular
 
-Authorized users are able to interrogate the Event Store (Immutable Datastore) for details of audit-worthy events.
+Authorized users are able to interrogate the Auditing BC via an Operation API exposed for this purpose for details of audit-worthy events.
 
 ## Functional Overview
 
-![Use Case - Auditing System Functional Overview](./assets/Mojaloop2RefArch_FunctionalOverview_09202021.png)
+![Use Case - Auditing System Functional Overview](./assets/ML2RA_FunctionalOverview_09282021.png)
```
Comment (mdebarros) [inline docs/refarch/boundedContexts/auditing/index.md:15]: Does this image seem to have alot of empty space on the right side, or is it just me?

File: docs/refarch/boundedContexts/platformConfigAndSecretMan/index.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
+# Platform Configuration And Secret Management BC
+
+{overview}
```
Comment (mdebarros) [inline docs/refarch/boundedContexts/platformConfigAndSecretMan/index.md:3]: @pedrosousabarreto would you be so kind to expand on this.

**Review by mdebarros — CHANGES_REQUESTED** [review]
Looks good @mwmentor, some minor suggestions.

I have also asked @koekiebox and @pedrosousabarreto to participate.

File: docs/refarch/boundedContexts/accountLookupAndDiscovery/index.md [hunk 5]
```diff
@@ -55,49 +57,12 @@ Get transfer/s Participant/s details.
 
 #### Description
 
-Get transfer/s Party/s details.
+Where a participant DFSP queries another participant DFSP for the details of a Party which the second DFSP holds, this UC is used to validate the request and provide the requested Party data to the requesting DFSP.
```
Comment (koekiebox) [inline docs/refarch/boundedContexts/accountLookupAndDiscovery/index.md:60]: If we opt for `Use Case` vs `UC`, we also need to change it here to be consistent throughout.

File: docs/refarch/boundedContexts/platformConfigAndSecretMan/index.md [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+> Configuration & Secret Bootstrapping
+
+### Perform Transfer (universal mode)
+
+#### Description
+
+#### Flow Diagram
+
+![Use Case - Example REPLACE ME](./assets/useCaseExample.png)
+> example image - replace
```
Comment (koekiebox) [inline docs/refarch/boundedContexts/platformConfigAndSecretMan/index.md:31]: Should this be replaced as part of this PR? Couple of occurrences in this file.

File: docs/refarch/boundedContexts/scheduling/index.md [hunk 7]
```diff
@@ -1,23 +1,133 @@
...
 
 | Term | Description |
 |---|---|
-| Term1 | Description1 |
+| Client BC | Any other BC making use of the Scheduling BC Services |
 
 ## Use Cases
 
-### Perform Transfer (universal mode)
+***Note:*** *An "Update Reminder" UC has not been provided. The recommended course of action to follow is to Delete and Create a new Reminder.*
```
Comment (koekiebox) [inline docs/refarch/boundedContexts/scheduling/index.md:19]: Would we lose any auditing/associated data when deleting a reminder vs doing an update _(at least that we care about)_?

**Review by koekiebox — CHANGES_REQUESTED** [review]
Minor queries/suggestions.

File: docs/refarch/boundedContexts/accountLookupAndDiscovery/index.md (hunk 5 above)
Comment (mdebarros) [reply docs/refarch/boundedContexts/accountLookupAndDiscovery/index.md:60]: It is defined as a Common Term --> https://ref-arch-docs.moja-lab.live/refarch/boundedContexts/intro.html

So I am ok to keep it as long as we are consistently using it.

File: docs/refarch/boundedContexts/platformConfigAndSecretMan/index.md (hunk 6 above)
Comment (mwmentor) [reply docs/refarch/boundedContexts/platformConfigAndSecretMan/index.md:31]: This file is part of the template sample that we have used to try and introduce a semblance of uniformity to the look and feel of the document.  I will be doing some tweaks in the coming days to the look and feel of the document and either replacing or hiding the "blank" templates wherever I find them.  Thanks :-)

**Review by koekiebox — APPROVED** [review]
LGTM.

**Comment by pedrosousabarreto** [comment]
Most of these changes have been cherry picked to other PRs. Closing.

**Pull request closed without merging**